apps/sms/views/conversation/style/composer.css

```diff
@@ -4,12 +4,6 @@
   background: #e1f0ec;
```

Comment:
As far as I understood from your bugzilla comment ```composer.css``` will be for composer widget/module only that is why I thought we can move view-related selectors out of this file, but maybe I missed something?